app/controllers/apply_forms_controller.rb

```diff
@@ -0,0 +1,73 @@
+class ApplyFormsController < ApplicationController
+	before_action :set_apply_form, only: [:show, :edit, :update, :destroy]
+
+  # GET /apply_form
+  def index
+    @apply_form = ApplyForm.all
```

Comment:
are we going to let the user to see all apply forms? this must be done at admin side.